der/src/asn1/octet_string.rs

```diff
@@ -113,6 +113,30 @@ impl<'a> TryFrom<&&'a [u8]> for OctetStringRef<'a> {
     }
 }
 
+/// Hack for simplifying the custom derive use case.
```

Comment:
I don't think this is a "hack", it's a reasonable conversion to allow encoding/decoding byte arrays as `OCTET STRING`